drivers/hud_apr/app/models/hud_apr/generators/shared/fy2020/base.rb

```diff
@@ -467,6 +467,7 @@ def needs_ce_assessments?
     # race field. The order that the races appear in the report is encoded in the 'order' of this hash.
     # This practice is very brittle, so we'll copy those here and hard code those relationships
     private def races
+      TodoOrDie('When we update reporting for 2022 spec', by: '2021-10-01')
```

Comment:
This is the 2020 generator, does it need to move?

Reply:
Removed